src/devices/tuya.ts

```diff
@@ -816,7 +816,7 @@ const definitions: Definition[] = [
         model: 'TS0203',
         vendor: 'Tuya',
         description: 'Door sensor',
-        fromZigbee: [fz.ias_contact_alarm_1, fz.battery, fz.ignore_basic_report, fz.ias_contact_alarm_1_report],
+        fromZigbee: [fz.ias_contact_alarm_1, fz.batteryIgnoringPercentage, fz.ignore_basic_report, fz.ias_contact_alarm_1_report],
```

Comment:
This ends up doing this for all models, not just the ones I've tested myself. Is that OK?